pkg/audit/manager_test.go

```diff
@@ -101,27 +101,42 @@ func Test_auditFromCache(t *testing.T) {
 
 	_, err = client.AddTemplate(context.Background(), fakes.DenyAllRegoTemplate())
 	require.NoError(t, err, "adding denyall constraint template")
-	_, err = client.AddConstraint(context.Background(), fakes.DenyAllConstraint())
-	require.NoError(t, err, "adding denyall constraint")
 
 	tests := []struct {
 		name            string
 		processExcluder *process.Excluder
+		constraint      *unstructured.Unstructured
 		wantViolation   bool
 	}{
 		{
 			name:            "obj excluded from audit",
 			processExcluder: processExcluderFor([]string{"test-namespace-1"}),
+			constraint:      fakes.DenyAllConstraint(),
 		},
 		{
 			name:            "obj not excluded from audit",
 			processExcluder: processExcluderFor([]string{}),
+			constraint:      fakes.DenyAllConstraint(),
+			wantViolation:   true,
+		},
+		{
+			name:            "audit excluded from constraint",
```

Comment:
webhook?

Reply:
can you clarify a little more here wdym? @sozercan 

This is a test for constraint without `audit` enforcementPoint.